internal/query/user.go

```diff
@@ -872,6 +877,7 @@ func scanUser(row *sql.Row) (*User, error) {
 	isPhoneVerified := sql.NullBool{}
 	passwordChangeRequired := sql.NullBool{}
 	passwordChanged := sql.NullTime{}
+	mfaInitSkipped := sql.NullTime{}
```

Comment:
`scanUser` is called by multiple functions, but only `GetUserByID` selects the new column right now. It^s missing at least in `GetUser` and `GetUserByLoginname`

Reply:
added column to the other uses. Waiting for the pipeline to confirm nothing remains broken.